docs/debugging/thundershark.md

```diff
@@ -0,0 +1,82 @@
+## Overview
+
+*ThunderShark* is a tool for analyzing COM-RPC traffic. It allows inspecting the out-of-process communication in a human-friendly presentation. 
+
+In particular it aids: 
+- profiling - surveying call duration and frame sizes, 
+- debugging - inspecting the parameters and return values passed, 
+- flow analysis - investigating the order of calls and notifications, keeping track of interfaces instance's lifetime. 
+
+It can also have an educational value - by showing their internals dissected it lets developers examine how COM frames are built.

```

Comment:
It can also have an educational value by showing their internals dissected. It lets developers examine how COM frames are built.

Reply:
rephrase a bit? he it after dissected is a bit unexpected..

Reply:
Perhaps there should've been a collon instead of a pause... "by showing their internals dissected it lets developers examine how COM frames are built", I think this is valid English :)